draft-ietf-quic-transport.md

```diff
@@ -2254,11 +2254,11 @@ abandons its attempt to validate the path.
 Endpoints SHOULD abandon path validation based on a timer. When setting this
 timer, implementations are cautioned that the new path could have a longer
 round-trip time than the original.  A value of three times the larger of the
-current Probe Timeout (PTO) or the initial timeout (that is, 2*kInitialRtt) as
-defined in {{QUIC-RECOVERY}} is RECOMMENDED.  That is:
+current Probe Timeout (PTO) or the PTO for the new path (that is, using
+kInitialRtt as defined in {{QUIC-RECOVERY}}) is RECOMMENDED.  That is:
 
 ~~~
-   validation_timeout = max(3*PTO, 6*kInitialRtt)
+   validation_timeout = 3*max(PTO_current, PTO_new)
```

Comment:
PTO_current and PTO_new aren't defined, so even thought I understand the intent, I'm no longer sure this pseudocode is helpful.  Possibly it'd be better to remove it?

Reply:
That is probably good.  I will do that.